packages/slider/stories/Slider.stories.tsx

```diff
@@ -27,7 +27,7 @@ export const Disabled = () => {
     <div>
       <output>{value}</output>
       <Slider
-        onChange={(value) => setValue(value)}
+        onChange={(val) => setValue(val)}
```

Comment:
I wouldn't mind adding https://github.com/fabric-ds/react/pull/187/files#diff-0be3b4850e403dc499c4a02e50cdde7ceb13fc901a97da54b82150cddfd867c1R20 in this story to be able to easily test the fix.

Reply:
yes nice!